Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM python:3.7
+FROM python:3.10
 
 WORKDIR /app
 
-RUN useradd dpl && \
-    pip install --pre poetry && \
+RUN useradd --create-home dpl && \
+    pip install "poetry>=1.1.2,<2" && \
```

Comment:
I had trouble with Python 3.10 + Poetry versions older than 1.1.12. Could be worth specifying it as the minimum here?

Reply:
I meant to do this, actually, this is a typo :facepalm: